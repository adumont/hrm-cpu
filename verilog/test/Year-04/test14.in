47 f3
55 4f
34 52
53 2d
03 c7
19 ec
01 28
37 c1
28 5e
20 80
10 2a
06 55
06 17
48 ef
18 37
54 a0
09 62
27 b3
18 9d
58 6a
58 44
42 f8
16 b0
41 69
46 9c
25 39
17 9b
26 ee
40 b6
18 1a
45 81
24 07
21 71
10 9a
20 93
35 f8
55 df
43 d0
40 49
40 e9
11 b4
28 39
54 48
