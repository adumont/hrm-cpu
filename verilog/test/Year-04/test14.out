4f
f3
2d
52
ec
c7
c1
28
80
5e
55
2a
ef
17
a0
37
b3
62
6a
9d
f8
44
69
b0
39
9c
ee
9b
1a
b6
07
81
9a
71
f8
93
d0
df
e9
49
39
b4
